validator/length.go

```diff
@@ -0,0 +1,97 @@
+package validator
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+type LengthRule struct {
+	Use     Limit
+	Short   Limit
+	Long    Limit
+	Example Limit
+}
+
+type Limit struct {
+	Min, Max int
+}
+
+func (l *LengthRule) Validate(cmd *cobra.Command) []error {
+
+	var errors []error
+	err := validateCobraCommand(cmd, l)
+	errors = append(errors, err...)
+
+	for _, child := range cmd.Commands() {
+		err := validateCobraCommand(child, l)
+		errors = append(errors, err...)
+
+	}
+
+	return errors
+}
+
+func validateCobraCommand(cmd *cobra.Command, l *LengthRule) []error {
+	var errors []error
+
+	cmdPath := cmd.CommandPath()
+
+	use := cmd.Use
+	useErr := validateField(l.Use, use, cmdPath)
```

Comment:
This part might bring some maintenance challenges and breaks DRY rule.

Reply:
We can use reflect package to get values by strings there is no need to repeat them